prov/rxm/src/rxm.h

```diff
@@ -404,12 +405,14 @@ struct rxm_conn {
 	struct fid_ep *msg_ep;
 	struct rxm_send_queue send_queue;
 	struct dlist_entry postponed_tx_list;
+	struct rxm_buf_pool rx_buf_pool;
 	struct dlist_entry posted_rx_list;
 	struct util_cmap_handle handle;
 	/* This is saved MSG EP fid, that hasn't been closed during
 	 * handling of CONN_RECV in CMAP_CONNREQ_SENT for passive side */
 	struct fid_ep *saved_msg_ep;
 	struct dlist_entry saved_posted_rx_list;
+	struct dlist_entry tx_entry_for_release;
```

Comment:
Basically, can we drop the changes related to this and just keep the entries on the posted list, with the main thread handling them?